### ISSUE TITLE
Cannot open files inside VS code

### ISSUE THREAD

**Issue by lopar6** [microsoft/vscode#126141]
Issue Type: <b>Bug</b>

In the file explorer, i cannot double click to open files.
The only way that works is right clicking and choosing "open with"

VS Code version: Code 1.56.2 (054a9295330880ed74ceaedda236253b4f39a335, 2021-05-12T16:45:26.313Z)
OS version: Linux x64 5.4.0-73-generic snap

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4200)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|1, 2, 2|
|Memory (System)|15.58GB (9.18GB free)|
|Process Argv|--force-user-env --no-sandbox --unity-launch --no-sandbox --crash-reporter-id 67e73194-dcea-4498-a059-562599d602f3|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|cinnamon|
|XDG_CURRENT_DESKTOP|X-Cinnamon|
|XDG_SESSION_DESKTOP|cinnamon|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|2.1.0
vscode-css-formatter|aes|1.0.1
vscode-django|bat|1.6.0
dart-code|Dar|3.23.0
flutter|Dar|3.23.0
python-extension-pack|don|1.6.0
gitlens|eam|11.4.1
vscode-html-css|ecm|1.10.2
smartmulticursor|ida|0.0.3
vscode-nuget-package-manager|jmr|1.1.6
MagicPython|mag|1.1.0
git-graph|mhu|1.30.0
vscode-docker|ms-|1.13.0
csharp|ms-|1.23.12
python|ms-|2021.5.926500501
vscode-pylance|ms-|2021.6.1
jupyter|ms-|2021.6.832593372
remote-containers|ms-|0.177.2
cpptools|ms-|1.4.1
vsliveshare|ms-|1.0.4419
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
java|red|0.79.2
LiveServer|rit|5.6.1
vscodeintellicode|Vis|1.2.14
vscode-java-debug|vsc|0.34.0
vscode-java-dependency|vsc|0.18.4
vscode-java-pack|vsc|0.16.0
vscode-java-test|vsc|0.30.0
vscode-maven|vsc|0.31.0
jinja|who|0.0.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspyt653:30270858
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833:30267464
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30286553
pythonvsuse255cf:30319632
vscgsv2:30307504
vscorehov:30309549
vscod805cf:30301675
binariesv517:30312825

```

</details>

**Comment by gjsjohnmurray** [comment]
Please use the Start Extension Bisect command to investigate whether the problem is being caused by one of your extensions.

**Comment by gtaifu** [comment]
Same problem. I have used Bisect, and the same problem exists after all extensions have been disabled. So Bisect told me this is a problem of Code.

**Comment by bpasero** [comment]
Maybe another editor associations issue?

**Comment by lramos15** [comment]
Can you please post your settings.json?

**Comment by lopar6** [comment]
I have reinstalled and have not been able to reproduce the bug. I was unable to locate any other problems. The reinstall fixed whatever the problem was.

**Comment by varora1406** [comment]
Same problem
VScode - 1.56.2
System - MacOS

Settings JSON - 
`{
    "todo-tree.tree.showScanModeButton": false,
    "workbench.iconTheme": "vscode-icons",
    "editor.suggestSelection": "first",
    "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
    "git.autofetch": true,
    "terminal.integrated.shell.windows": "C:\\WINDOWS\\System32\\cmd.exe",
    "git.confirmSync": false,
    "workbench.startupEditor": "newUntitledFile",
    "[typescriptreact]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "[javascript]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "editor.formatOnSave": true,
    "[typescript]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "[jsonc]": {
        "editor.defaultFormatter": "vscode.json-language-features"
    },
    "[json]": {
        "editor.defaultFormatter": "rvest.vs-code-prettier-eslint"
    },
    "vs-kubernetes": {
        "vs-kubernetes.kubectl-path.linux": "/home/vaibhav/.vs-kubernetes/tools/kubectl/kubectl",
        "vs-kubernetes.minikube-path.linux": "/home/vaibhav/.vs-kubernetes/tools/minikube/linux-amd64/minikube",
        "vs-kubernetes.helm-path.linux": "/home/vaibhav/.vs-kubernetes/tools/helm/linux-amd64/helm"
    },
    "[yaml]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "typescript.updateImportsOnFileMove.enabled": "always",
    "git.suggestSmartCommit": false,
    "editor.linkedEditing": true,
    "[html]": {
        "editor.defaultFormatter": "rvest.vs-code-prettier-eslint"
    },
    "[scss]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "angular.enable-experimental-ivy-prompt": false,
    "tabnine.experimentalAutoImports": true,
    "workbench.editorAssociations": {
        "*.ipynb": "jupyter.notebook.ipynb"
    },
    "javascript.updateImportsOnFileMove.enabled": "always",
    "emmet.showSuggestionsAsSnippets": true,
    "emmet.triggerExpansionOnTab": true,
    "[javascriptreact]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "redhat.telemetry.enabled": false,
    "diffEditor.ignoreTrimWhitespace": false
}`

**Comment by varora1406** [comment]
Fixed for me when updated to 1.57
